model/job/redis_scheduler_test.go

```diff
@@ -141,8 +141,18 @@ func TestRedisSchedulerWithTimeTriggers(t *testing.T) {
 	for _, trigger := range ts {
 		switch trigger.Infos().TID {
 		case atID:
+			assert.True(t, tat.Infos().Metadata.CreatedAt.Equal(trigger.Infos().Metadata.CreatedAt))
```

Comment:
[nit] I prefer `assert.Equal` to `assert.True(t, equalCondition)` as it gives a more helpful message when the assertion fails

Reply:
It seems that redis does not handle the monotonic clock reading (`m=...`), which leads to a not equal time and a failing test